ngraph/frontend/frontend_manager/include/frontend_manager/extension.hpp

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2021 Intel Corporation
```

Comment:
Should this file be called `telemetry_extension` or will it perhaps contain more classes in the future?

Reply:
I guess we'll put some extension classes here, but I'm not sure yet.
I will rename in the next PR when we finally decide.

Reply:
okay, I will rename

Reply:
done